### PR TITLE
Signing support for stargate

### LINKED ISSUE #41: Allow executing contract for stargate
The current contract executing funcionality is only enabled for Launchpad, we should add support for executing contracts in Stargate.

### PR DESCRIPTION
**Pull request by abefernan**
Closes #41

To do:
- missing amino types for placeholder in Stargate
- improve context state management (userAddress and signingClient, mainly)

### CODE REVIEW

File: src/ui-utils/clients.ts [hunk 1]
```diff
@@ -23,27 +29,35 @@ export function isLaunchpadClient(
...
+): signingClient is LaunchpadSigningClient {
+  return signingClient instanceof LaunchpadSigningClient;
+}
+
 export function generateMnemonic(): string {
   return Bip39.encode(Random.getBytes(16)).toString();
 }
 
-export function loadOrCreateMnemonic(): string {
+export function loadOrCreateMnemonic(mnemonic?: string): string {
```
Comment (webmaster128) [inline src/ui-utils/clients.ts:48]: This should be called something like `importOrGenerateMnemonic` as it does not load anymore

File: src/ui-utils/clients.ts [hunk 2]
```diff
@@ -23,27 +29,35 @@ export function isLaunchpadClient(
...
   const key = "burner-wallet";
-  const loaded = localStorage.getItem(key);
```
Comment (webmaster128) [inline src/ui-utils/clients.ts:32]: If the mnemonic is never loaded, this user gets a new one every time the page is reloaded. This seems to be annoying for testing. Do you have a specific reason from removing the load functionality?

Comment (abefernan) [reply src/ui-utils/clients.ts:32]: It was a naive way of avoiding that the user got stuck with an account without funds. I just pushed some changes that keep the account stored, but if the user enters another mnemonic it replaces it.

Comment (webmaster128) [reply src/ui-utils/clients.ts:32]: > but if the user enters another mnemonic it replaces it

Yeah that's good

**Review by webmaster128 — APPROVED** [review]

**Pull request merged**